docs/getting-started/certificate-pinning.adoc

```diff
@@ -0,0 +1,46 @@
+== Certificate Pinning
+
+Certificate pinning can be enabled in individual SDKs through the `mobile-services.json` file.
+Certificate pinning will only be enabled for services which are used directly by the SDKs. External
+services must have pinning configured separately. For more information on setting up certificate
+pinning for external services see the
+link:https://developer.android.com/training/articles/security-config.html#CertificatePinning[Android network security guide].
+
+=== Generating Pinning Configuration
+
+The Android SDK gets its configuration from the `https` section of the `mobile-services.json` file
+in a project.
+
+[source,json]
+----
+{
+    "services":[],
+    "https": [{
+        "host": "example.sync.service",
+        "certificateHash": "exampleHash"
+    }]
+}
+----
+
+To include the `https` section in configuration when generating it using the Mobile CLI use the
+`--include-cert-hashes` flag when retrieving a client configuration. By default, self-signed or
+invalid certs will not be permitted to be included in the certificate pinning configuration. To
+allow these to be included use the `--allow-invalid-certs` flag.
```

Comment:
can we reuse this flag to support self-signed certs as described in this [proposal](https://github.com/aerogear/proposals/pull/34)?

Reply:
Yes I think we can. I can rename this to be `--allow-self-signed` or something if that suits both a bit better?

Reply:
Yeah, I think I prefer `--allow-self-signed-certs`